src/mappings/event/vault.ts

```diff
@@ -158,3 +166,90 @@ export async function decreaseLockedCollateral(
         })
     );
 }
+
+export async function withdrawCollateralVault(
+    ctx: Ctx,
+    block: SubstrateBlock,
+    item: EventItem,
+    entityBuffer: EntityBuffer
+): Promise<void> {
+    const rawEvent = new VaultRegistryWithdrawCollateralEvent(
+        ctx,
+        item.event
+    );
+    let e;
+    let vaultId;
+    if (rawEvent.isV6) {
+        e = rawEvent.asV6;
+        vaultId = encodeLegacyVaultId(e.vaultId);
+    }
```

Comment:
please avoid line breaks before the `else` part to keep the code style consistent.
ie.
```
if (/* condition */) {
  // logic
} else if (/* other condition */) {
  // other logic
} else {
  // more logic
}
```

Reply:
Fixed!